Dockerfile.web

```diff
@@ -5,7 +5,7 @@
 # the cpu pool size to something realistic.
 ARG NODE_OPTIONS="--v8-pool-size=4"
 
-# PHASE 1 - build static html.
+# PHASE 1 - build frontend.
 # Pull from local registry - we can't pull from docker due to limits.
 # see https://catalog.redhat.com/software/containers/ubi8/nodejs-14/5ed7887dd70cc50e69c2fabb for
```

Comment:
```suggestion
# see https://catalog.redhat.com/software/containers/ubi9/nodejs-20/64770ac7a835530172eee6a9 for
```

Reply:
Sorry, missed this stale comment in the previous PR. Would you mind slipping in this update?